.github/workflows/migrate.yml

```diff
@@ -36,7 +36,7 @@ jobs:
       - name: Set flag
         id: set-flag
         run: |
-          echo "SHOULD_RUN=steps.should-run.outputs.should_run" >> $GITHUB_OUTPUT
+          echo "SHOULD_RUN=steps.should-run.outputs.should-run" >> $GITHUB_OUTPUT
 
   migrate:
     needs: [should-run]
```

Comment:
I don't think we can make the individual matrix jobs required if they only run sometimes. Might be easier to make a post-migrate job required.

```
post-migrate:
    needs: [migrate]
    if: always() && !failure() && !cancelled()
    runs-on: ubuntu-latest
    steps:
    - name: Success
        run: echo success
```

Reply:
Never tried this, but I think it might work.

Reply:
Yeah, the matrix jobs screw it up. I noticed it last week with the initial PR on this.